include/dlr.h

```diff
@@ -388,6 +390,56 @@ int SetDLRCustomAllocatorFree(DLRFreeFunctionPtr custom_free_fn);
 DLR_DLL
 int SetDLRCustomAllocatorMemalign(DLRMemalignFunctionPtr custom_memalign_fn);
 
+/*
+ * \brief Creates a DLR model for TVM GraphRuntime models. Accepts graph.json and params data
+ *        from memory rather than loading from files on disk. lib.so must still be on disk
+ *        since dlopen will be used to load it.
+ * \param handle The pointer to save the model handle.
+ * \param graph String of loaded graph.json file
+ * \param lib_path Path to the TVM lib.so file on disk
+ * \param params String of binary data loaded from TVM params file
+ * \param params_len length of the params string
```

Comment:
Because we are asking users to provide `params_len` I guess the array of chars might not have terminating null character.
In that case we should not call it "String".  It is just a byte array.

Reply:
thats right, it may not be null terminated, and in fact might have "null characters" in the middle since its just binary data.